posts/schrodingers-pineapple-upside-down-cake-front-end-development-for-back-end-engineers.md

```diff
@@ -0,0 +1,371 @@
+---
+authors:
+- Christina Annas
+tags:
+-back-end
+-career-development
+-front-end
+-javascript
+date: 2022-02-28T22:00:00.000Z
+title: "Schrödinger's Pineapple Upside Down Cake: Front-End Development for Back-End Engineers"
+---
+
+Hey, good to see you again! Let's start by reviewing where we left off. We have a webpage that displays weather information for each of the Ippon offices in the US. However, although our page is created in a much more dynamic way than how it started out, we are still displaying out-of-date information. 
+
+```
+todo: add in from-last-time screenshot here.
```

Comment:
Missing a screenshot here @christinaannas? You mention it in a todo but don't see an image link :)

Reply:
Oh my gosh *brilliant* thank you!